README.md

```diff
@@ -691,6 +691,7 @@ API | Description | Auth | HTTPS | CORS |
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [BikeWise](https://www.bikewise.org/documentation/api_v2) | Bikewise is a place to learn about and report bike crashes, hazards and thefts | No | Yes | Unknown |
+| [Canadian Football League (CFL)](http://api.cfl.ca/) | Official JSON API providing real-time league, team and player statistics about the CFL | `apiKey` | Yes | No |
```

Comment:
Please remove "JSON API" from the description.

Reply:
With lots of APIs listed being an unofficial source, I thought it'd be helpful to note that this one is direct from the league. "Official providing real-time league..." doesn't make sense as a sentence?

Reply:
Sure, thanks for clarifying.